LandSandBoat/server
# [geo] scroll scripts, aura effects and spell scripts

**_I affirm:_**

## What does this pull request do?

Adds the ramining spell scripts,
Adds the scripts for the indi/geo effects that are missing,
Adds the scripts for the indi scrolls, so they can used to learn indi spells.

## Steps to test these changes

Used each spell and tested potency for each one at various skill levels to confirm they were correct, as well as the effect they granted.

Added each indi scroll and used each one to confirm they added the correct spell.
